src/common.jl

```diff
@@ -1,3 +1,5 @@
+using DataStructures
```

Comment:
This is an unrelated change. To keep the history clean we'd better to revert this.